docs/AUTHENTICATION.md

```diff
@@ -13,9 +13,23 @@
 
 The following environment variables are required for firebase:
 
-- `REACT_APP_FIREBASE_PROJECT_ID`
-- `REACT_APP_FIREBASE_API_KEY`
+| Name                            | Description                                                                                       | Required                         |
+| ------------------------------- | ------------------------------------------------------------------------------------------------- | -------------------------------- |
+| `REACT_APP_FIREBASE_PROJECT_ID` | Firebase is used for basic authentication this environment and the two below are needed for this. | Yes If authentication is enabled |
+|                                 | The project ID for your Firebase project. Can be obtained from your Firebase project              |                                  |
+| `REACT_APP_FIREBASE_API_KEY`    | The api key for your Firebase project. Can be obtained from your Firebase project                 | Yes If authentication is enabled |
```

Comment:
Don't think authentication can ever not be enabled now, so these to I think are now just required?

Reply:
I have changed authentication required to Yes.